### PR TITLE
feat(pds-ember): add Nav component

### PR DESCRIPTION
**Pull request by CITguy**
* port applicable styles from HCP
* update `<Pds::App>` stories with nav component in place

JIRA:
* PDS-107
* PDS-174
* PDS-175

## Screenshots
pds-nav -- states

### CODE REVIEW

File: packages/pds-ember/addon/components/pds/nav/docs.mdx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { ArgsTable } from '@storybook/addon-docs/blocks';
+export const TITLE = 'Components / Nav';
+
+# Nav
+
+TBD...
```
Comment (CITguy) [inline packages/pds-ember/addon/components/pds/nav/docs.mdx:6]: - [ ] Update Docs

Comment (CITguy) [inline packages/pds-ember/addon/components/pds/nav/docs.mdx:6]: - [ ] Update Docs

File: packages/pds-ember/app/styles/pds/components/link/__config.scss [hunk 2]
```diff
@@ -10,24 +10,43 @@ $_module: "PDS.Components.Link";
...
+// _current route_.  However, this can be confusing alongside the :active
+// pseudo-class, which indicates that the link is actively being _pressed_.
+//
+// Rather than override every `<LinkTo>`, we'll embrace the default behavior.
+// This mixin is provided to declaratively apply "current" styling in Sass
+// stylesheets, using the expected semantic naming.
+//
+// See https://api.emberjs.com/ember/release/classes/Ember.Templates.components/methods/LinkTo#handling-current-route
+@mixin whenCurrent {
+  /* [debug] #{$_module}@apply */
```
Comment (CITguy) [inline packages/pds-ember/app/styles/pds/components/link/__config.scss:44]: update comment

File: packages/pds-ember/app/styles/pds/components/nav/index.scss [hunk 3]
```diff
@@ -0,0 +1,8 @@
+@use "__config" as *;
+
+@include apply {
+  @include layout;
+  @include appearance;
+}
+
+// NOTE: excluded `hr` styling from ported CSS
```
Comment (CITguy) [inline packages/pds-ember/app/styles/pds/components/nav/index.scss:8]: remove comment

**Review by meirish — APPROVED** [review]
Looks good!

**Pull request merged**